animation_url/vite.config.ts

```diff
@@ -14,6 +14,11 @@ const config: UserConfig = {
       allow: [".."],
     },
   },
+  optimizeDeps: {
+    exclude: [
+      "@tableland/sqlparser"
```

Comment:
This is a fix for the error:
```
Not found: /node_modules/.vite/deps/main.wasm
Error: Not found: /node_modules/.vite/deps/main.wasm
```
It would be nice to fix this elsewhere, but I can't find any other solutions.